src/test/java/org/sonar/scm/git/blame/RepositoryBlameCommandIT.java

```diff
@@ -37,6 +37,71 @@
 import static org.sonar.scm.git.GitUtils.moveFile;
 
 public class RepositoryBlameCommandIT extends AbstractGitIT {
+  @Test
+  public void uncommitted_delete_files_have_no_blame() throws GitAPIException, IOException {
+    createFile(baseDir, "fileA", "line1");
+    String c1 = commit("fileA");
+
+    deleteFile(baseDir, "fileA");
+
+    BlameResult result = blame.setFilePaths(Set.of("fileA")).call();
+    assertThat(result.getFileBlames()).extracting(FileBlame::getPath, FileBlame::getCommitHashes)
+      .isEmpty();
+  }
+
+  @Test
+  public void uncommitted_renamed_files_have_no_blame() throws IOException, GitAPIException {
+    createFile(baseDir, "fileA", "line1");
+    String c1 = commit("fileA");
+
+    deleteFile(baseDir, "fileA");
+    createFile(baseDir, "fileB", "line2");
```

Comment:
shouldn't it be "line1" to match a rename?

Reply:
yes, good catch.